packages/bierzo-wallet/src/routes/transactions/components/DownloadCSV/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { Fab, makeStyles, Theme } from '@material-ui/core';
+import { useTheme } from '@material-ui/styles';
+import Block from 'medulas-react-components/lib/components/Block';
+import CircleImage from 'medulas-react-components/lib/components/Image/CircleImage';
+import Typography from 'medulas-react-components/lib/components/Typography';
+import * as React from 'react';
+
+import { getBorderColor } from '../../../../theme/css';
+import download from '../../assets/download.svg';
+
+export const CSV_PADDING = 20;
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    width: 'auto',
+    justifyContent: 'left',
+    textTransform: 'inherit',
+  },
+  sizeSmall: {
+    height: `${theme.spacing(4)}px`,
+    padding: '0 !important',
+  },
+  text: {
+    padding: `0px ${theme.spacing(1.5)}px`,
```

Comment:
Nice, I thought that it should be possible to provide non integer (real) values but didn't tried.